Update naming conventions for both IaaS and PaaS

juvchan opened the pull request:
The full names for IaaS and PaaS should be Infrastructure as a Service and Platform as a Service based on The Gartner IT Glossary.

https://www.gartner.com/it-glossary/infrastructure-as-a-service-iaas/
https://www.gartner.com/it-glossary/platform-as-a-service-paas/

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Update naming conventions for both IaaS and PaaS

The full names for IaaS and PaaS should be Infrastructure as a Service and Platform as a Service based on The Gartner IT Glossary.

https://www.gartner.com/it-glossary/infrastructure-as-a-service-iaas/
https://www.gartner.com/it-glossary/platform-as-a-service-paas/
